protocol/vm/ops.go

```diff
@@ -201,19 +201,20 @@ const (
 	OP_TXSIGHASH     Op = 0xae
 	OP_BLOCKHASH     Op = 0xaf
 
-	OP_CHECKOUTPUT   Op = 0xc1
-	OP_ASSET         Op = 0xc2
-	OP_AMOUNT        Op = 0xc3
-	OP_PROGRAM       Op = 0xc4
-	OP_MINTIME       Op = 0xc5
-	OP_MAXTIME       Op = 0xc6
-	OP_TXREFDATAHASH Op = 0xc7
-	OP_REFDATAHASH   Op = 0xc8
-	OP_INDEX         Op = 0xc9
-	OP_OUTPUTID      Op = 0xcb
-	OP_NONCE         Op = 0xcc
-	OP_NEXTPROGRAM   Op = 0xcd
-	OP_BLOCKTIME     Op = 0xce
+	OP_CHECKOUTPUT Op = 0xc1
+	OP_ASSET       Op = 0xc2
+	OP_AMOUNT      Op = 0xc3
+	OP_PROGRAM     Op = 0xc4
+	OP_MINTIME     Op = 0xc5
+	OP_MAXTIME     Op = 0xc6
+	OP_TXDATAHASH  Op = 0xc7
```

Comment:
As far as I'm concerned, the naming of these opcodes (and the data fields they refer to) is not yet a settled question. (See https://github.com/chain/chain/pull/728#discussion_r105473953.) Although populating them with hashes is likely to be a common use case, nothing requires them to be hashes. I'd sooner keep the "ref" in the name and lose the "hash."

In any case, I want the code to reflect the names in the spec. No sense letting them diverge for the sake of saving a few diff lines in one PR.